Snakefile

```diff
@@ -89,6 +88,27 @@ rule parse_background_seqmeta:
             --fields {params.fasta_fields}
         """
 
+rule download_seattle_metadata:
+    message:
+        """
+        download_seattle_metadata: Downloading metadata for Seattle sequences
+        from ID3C
+        """
+    output:
+        metadata = "data/seattle_metadata.tsv"
+    params:
+        id3c_url = os.environ['ID3C_METADATA_URL'],
```

Comment:
I think it will be better to do `urljoin(os.environ["ID3C_URL"], "/v1/shipping/augur-build-metadata")` instead of expecting the full endpoint in the environment variable. The code in this repo is written to the expectations of that particular `/v1` endpoint and so it's good to explicitly define that here rather than make it implicit.